pages/common/git-rebase.md

```diff
@@ -17,3 +17,7 @@
 - Abort a rebase in-progress:
 
 `git rebase --abort`
+
+- Rebase your local branch by specifying new base commit and old base commit:
+
+`git rebase --onto {{NEW-BASE-COMMIT}} {{OLD-BASE-COMMIT}}`
```

Comment:
I'd love to :smiley: 
Maybe it's a whole-scan through `*.md` files.
And let me confirm that the token syntax guide is [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax), right?